package.json

```diff
@@ -4,16 +4,15 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "coverage": "npm run db-migrate && mocha --reporter mochawesome --timeout=20000 --require babel-register --exit",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
     "db-migrate": "sequelize db:migrate",
     "db:undo-migration": "sequelize db:migrate:undo:all",
     "lint": "eslint routes models config test index.js",
-    "test": "npm run db:undo-migration && npm run db-migrate && mocha test/index.js --timeout=20000 --require babel-register --exit",
-    "start": "sequelize db:migrate && node dist/index",
+    "start": "sequelize db:migrate && npm run build && node dist/index.js",
```

Comment:
@kleva-j should we be running the build script here?

Reply:
I wanted to get the new changes to the code to be transpiled when I run the app on development because babel-watch was not working well on my local machine.
I forgot to remove for production.